.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Summary
+
+<!-- What is this pull request for? Does it fix any issues? -->
+
+### Checklist
+
+- [ ] If code changes were made then they have been tested.
+    - [ ] I have updated the documentation to reflect the changes.
+- [ ] This PR fixes an issue.
```

Comment:
Some of these checkboxes could be solved with a "type" field:

**Type:** bug / feature / documentation

Reply:
Not sure how I feel about this one.

Reply:
That makes sense